doc/introduction/measurements.rst

```diff
@@ -66,6 +66,36 @@ You can also use list comprehensions, and other common Python patterns:
         qml.RY(y, wires=1)
         return [qml.expval(qml.PauliZ(i)) for i in range(2)]
 
+As an example of combined measurements, let us look at
+a Bell state :math:`(|00\rangle + |11\rangle)/\sqrt{2}`, prepared
+by a ``Hadamard`` and ``CNOT`` gate.
+
+.. code-block:: python
+
+    dev = qml.device("default.qubit", wires=2)
+
+    @qml.qnode(dev)
+    def circuit():
+        qml.Hadamard(wires=0)
+        qml.CNOT(wires=[0, 1])
+        return [qml.sample(qml.PauliZ(i)) for i in range(2)]
```

Comment:
Here, just as a suggestion, perhaps being more explicit could also work (as we have a specific number of wires that are sampled from)

Reply:
I quite like the list comprehension :laughing: I feel that it's more common for a user to know Python list comprehension but not realise it can be used in a QNode, than not knowing list comprehension at all.

But maybe there is a better example to show the list comprehension (like when the num wires is larger)

Reply:
The example directly above shows it, so I will go with @antalszava 's suggestion to simplify